src/CoreWf/Expressions/AndAlso.cs

```diff
@@ -10,7 +10,9 @@ namespace System.Activities.Expressions
 using System.Activities.DynamicUpdate;
 
     //[SuppressMessage(FxCop.Category.Naming, FxCop.Rule.IdentifiersShouldNotMatchKeywords, Justification = "Optimizing for XAML naming. VB imperative users will [] qualify (e.g. New [AndAlso])")]
+#pragma warning disable CA1716 // Identifiers should not match keywords
```

Comment:
Wouldn't [this](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.suppressmessageattribute?view=netframework-4.8) work? Or any other less painful syntax :)

Reply:
Ok, that was the initial code :) I think, just remove the pragma for now. Perhaps we should enable the attributes at some point.

Reply:
Removed